{
	"name": "CVE-2019-9029",
	"description": "An issue was discovered in libmatio.a in matio (aka MAT File I/O Library) 1.5.13. There is an out-of-bounds read with a SEGV in the function Mat_VarReadNextInfo5() in mat5.c.",
	"packages": [
		{
			"name": "libmatio",
			"releases": {
				"bookworm": {
					"status": "resolved",
					"repositories": {
						"bookworm": "1.5.23-1"
					},
					"fixed_version": "1.5.13-2",
					"urgency": "low"
				},
				"bullseye": {
					"status": "resolved",
					"repositories": {
						"bullseye": "1.5.19-2"
					},
					"fixed_version": "1.5.13-2",
					"urgency": "low"
				},
				"buster": {
					"status": "resolved",
					"repositories": {
						"buster": "1.5.13-3"
					},
					"fixed_version": "1.5.13-2",
					"urgency": "low"
				},
				"sid": {
					"status": "resolved",
					"repositories": {
						"sid": "1.5.23-1"
					},
					"fixed_version": "1.5.13-2",
					"urgency": "low"
				},
				"stretch": {
					"status": "open",
					"repositories": {
						"stretch": "1.5.9-1"
					},
					"urgency": "low",
					"nodsa": "Minor issue",
					"nodsa_reason": ""
				}
			}
		}
	],
	"scope": "local"
}